cmd/datamon/cmd/bundle_list.go

```diff
@@ -27,7 +27,12 @@ func applyBundleTemplate(bundle model.BundleDescriptor) error {
 var BundleListCommand = &cobra.Command{
 	Use:   "list",
 	Short: "List bundles",
-	Long:  "List the bundles in a repo, ordered by their bundle ID",
+	Long: `List the bundles in a repo, ordered by their bundle ID.
+
+This is analogous to the "git log" command. The bundle ID works like a git commit hash.`,
+	Example: `% datamon bundle list --repo ritesh-test-repo
+Using config file: /Users/ritesh/.datamon/datamon.yaml
```

Comment:
Is the config file output still printed?

Reply:
fixed this (reviewed through all examples)